### ISSUE TITLE
"Reshaping JSON with jq" is missing link to json object shown in examples

### ISSUE THREAD

**Issue by twitu** [programminghistorian/jekyll#2111]
If you are reporting a problem with a lesson, please try to provide us with the following information:

- The full title of the lesson
Reshaping JSON with jq

- The system you are using (Mac, Linux, Windows)
Mac

- Version numbers of the relevant software you are using
https://jqplay.org/ vesion 1.6 (current)

- The exact steps you took that caused the problem
The page runs filters on an example JSON object. However for the second example shown `.artObjects` there is no artObjects key in the mentioned json.

> Let’s consider the JSON for Rembrandt’s [Nightwatch in the Rijksmuseum](https://www.rijksmuseum.nl/nl/collectie/SK-C-5):

It will be good to have the JSON object used for the example available for the reader to play along.

**Comment by ZoeLeBlanc** [comment]
Thanks for letting us know about this bug @twitu . @svmelton would this be something the @programminghistorian/english-team or @programminghistorian/technical-team should investigate?

**Comment by anisa-hawes** [comment]
I think the example code block of JSON related to Rembrandt's _Nightwatch in the Rijksmuseum_ is solely provided to illustrate how `{}`, `id` keys, and `[]` operate. 

It is in the following section, titled [Lesson Setup](https://programminghistorian.org/en/lessons/json-and-jq#lesson-setup) that readers follow the lesson using the [sample JSON provided here](https://programminghistorian.org/assets/jq_rkm.json). 
 `artObjects` appears on Line 5 of the sample. I tested from here onwards, and at each stage, the raw output I retrieved, matched what is shown in the screenshots exactly.   

From my point of view, there is no Maintenance issue here. Are you happy for me to close this Issue, @svmelton?
